### PR TITLE
Disable the option for nginx persistence with more than one replica

### LINKED ISSUE #21: Error when multiple Nginx (replicaCount > 1)
**Version of Helm and Kubernetes**:
NA

**Which chart**:
artifactory-ha

**What happened**:
When setting nginx.replicaCount=2, and nginx.persistance.enabled=true, The second instance of Nginx fails to start because volume claim is already bound to first instance:

```
kubectl get pods | grep artifactory

artifactory-artifactory-ha-member-0                    1/1       Running    0          9m
artifactory-artifactory-ha-member-1                    1/1       Running    0          7m
artifactory-artifactory-ha-primary-0                   1/1       Running    0          9m
artifactory-nginx-69f8d97768-kmpjd                     0/1       Init:0/2   0          9m
artifactory-nginx-69f8d97768-l95vs                     1/1       Running    0          9m
```

```
kubectl get pvc | grep artifactory

artifactory-nginx                                                Bound     pvc-e899f622-ab99-11e8-b520-0a9fd9e2d170   5Gi        RWO            standard       10m
volume-artifactory-artifactory-ha-member-0                       Bound     pvc-e8b40093-ab99-11e8-ac7b-1237bc16f1bc   200Gi      RWO            standard       10m
volume-artifactory-artifactory-ha-member-1                       Bound     pvc-4291763c-ab9a-11e8-ac7b-1237bc16f1bc   200Gi      RWO            standard       7m
volume-artifactory-artifactory-ha-primary-0                      Bound     pvc-e8bb615f-ab99-11e8-ac7b-1237bc16f1bc   200Gi      RWO            standard       10m
```

```
kubectl describe pod artifactory-nginx-69f8d97768-kmpjd

[...]
Events:
  Type     Reason                 Age                From                                 Message
  ----     ------                 ----               ----                                 -------
  Warning  FailedScheduling       47s (x3 over 48s)  default-scheduler                    PersistentVolumeClaim is not bound: "artifactory-nginx" (repeated 12 times)
  Normal   Scheduled              45s                default-scheduler                    Successfully assigned artifactory-nginx-69f8d97768-kmpjd to ip-10-80-2-89.ec2.internal
  Normal   SuccessfulMountVolume  44s                kubelet, ip-10-80-2-89.ec2.internal  MountVolume.SetUp succeeded for volume "artifactory-artifactory-ha-token-6wv9c"
  Warning  FailedMount            44s                attachdetach-controller              AttachVolume.Attach failed for volume "pvc-e899f622-ab99-11e8-b520-0a9fd9e2d170" : "Error attaching EBS volume \"vol-0c875b4587a4178e4\"" to instance "i-04aaaeb9a03b14d7d" since volume is in "creating" state
  Warning  FailedAttachVolume     33s                attachdetach-controller              Multi-Attach error for volume "pvc-e899f622-ab99-11e8-b520-0a9fd9e2d170" Volume is already exclusively attached to one node and can't be attached to another
```

**What you expected to happen**:

It is not clear if and why persistent volumes should be used for NGinx. 
Can you precise the use of these volumes and when they should be used or not ?

What should be the expected behaviour ? Should multiple persistent volumes and persistant volume claims must be created or the same volume should be shared with appropriate accessMode ?

**How to reproduce it** (as minimally and precisely as possible):

helm install --name artifactory --set nginx.replicaCount=2 jfrog/artifactory-ha

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Add information about nginx persistence configuration

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #21, fixes #243

### CODE REVIEW

**Comment by danielezer** [comment]
@eldada I didn't change anything in the artifactory chart - just the readme. I don't think there's a reason to bump the version there

**Comment by rimusz** [comment]
@danielezer any change to the chart including the readme should bump the chart version.
Please resolve `stable/artifactory-ha/CHANGELOG.md` conflicts

**Comment by danielezer** [comment]
Done

**Comment by jainishshah17** [comment]
@danielezer  Any chance you can merge this with https://github.com/jfrog/charts/pull/250?

**Comment by danielezer** [comment]
@jainishshah17 I think these are two different things and should be merged separately. Why do you think I should join them together?

**Comment by jainishshah17** [comment]
@danielezer  You are right. I took a look again both are different Nginx issue. I thought both are related to nginx restart.

**Comment by danielezer** [comment]
Cool, thanks @jainishshah17

**Comment by danielezer** [comment]
@eldada Done

**Comment by danielezer** [comment]
@jainishshah17 can you please approve? I need an additional approval

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+<<<<<<< HEAD
```
Comment (jainishshah17) [inline stable/artifactory-ha/CHANGELOG.md]: @danielezer Can you fix this?

File: stable/artifactory-ha/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 apiVersion: v1
 name: artifactory-ha
 home: https://www.jfrog.com/artifactory/
+<<<<<<< HEAD
```
Comment (jainishshah17) [inline stable/artifactory-ha/Chart.yaml]: @danielezer Looks like merge conflict.

File: stable/artifactory/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,11 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
+<<<<<<< HEAD
```
Comment (jainishshah17) [inline stable/artifactory/Chart.yaml]: @danielezer same here

File: stable/artifactory-ha/CHANGELOG.md (hunk 1 above)
Comment (danielezer) [reply stable/artifactory-ha/CHANGELOG.md]: Whoops, sorry I didn't commit my conflict resolutions. Sorry about that

File: stable/artifactory-ha/Chart.yaml (hunk 2 above)
Comment (danielezer) [reply stable/artifactory-ha/Chart.yaml]: Whoops, sorry I didn't commit my conflict resolutions. Sorry about that

File: stable/artifactory/Chart.yaml (hunk 3 above)
Comment (danielezer) [reply stable/artifactory/Chart.yaml]: Whoops, sorry I didn't commit my conflict resolutions. Sorry about that

**Comment by danielezer** [comment]
@jainishshah17 fixed the conflicts, sorry about that.

**Pull request merged**